test/utils/describeConformance.js

```diff
@@ -165,9 +170,8 @@ export function testRootClass(element, getOptions) {
     expect(document.querySelectorAll(`.${classes.root}`).length).to.equal(1);
 
     // Test that classes prop works
-    setProps({ classes: { ...classes, root: `${classes.root} ${className}` } });
```

Comment:
We cannot use `setProps` where there is `wrapMount` as the props are being passed to the wrong element.

I am now setting the `classes` prop on the initial rendering of the component on line 161.